datadog/resource_datadog_screenboard.go

```diff
@@ -175,7 +175,7 @@ func resourceDatadogScreenboard() *schema.Resource {
 					Default:  true,
 				},
 				"precision": &schema.Schema{
-					Type:     schema.TypeString,
+					Type:     schema.TypeFloat,
```

Comment:
Precision can be `"*"` so this won't work

Reply:
do you have a timeline for the fix for `Precision` I hate that it can be a number sometimes and a string other times.... :shakefistatserver: